### PR TITLE
Support for localized fields in brick.

### PR DESCRIPTION
**Pull request by andreas-gruenwald**
* Correctly add localized fields in brick in configuration tree.
* Correctly interpret localized fields in brick in DefaultValue element.
* Provide possiblity to ask if default value is in relation with a localized field (helps users to decide if output still needs to be translated, or is already translated).

### CODE REVIEW

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -206,10 +206,26 @@ private function getFieldDefinition($attributeName, $objectClass) {
             $brickKey = $attributeParts[1];
         }
 
-        $def = $objectClass->getFieldDefinition($attributeName);
-
+        $def = null;
```
[the remark this answers is not in the document]
Comment (andreas-gruenwald) [reply src/Controller/AdminController.php]: Yes, there is an else missing, where the $def is properly initialized for the non-brick case.
Commit is added.

File: src/Controller/AdminController.php [hunk 2]
```diff
@@ -206,10 +206,26 @@ private function getFieldDefinition($attributeName, $objectClass) {
...
+        $brickInfos = null;
+        if ($brickKey && strpos($brickType, '?') === 0) {
```
[the remark this answers is not in the document]
Comment (andreas-gruenwald) [reply src/Controller/AdminController.php]: I added the block as suggested by Josef, trying to break as less downward compatibility as possible. 
We have to support the old case where the attribute parts are separated with "~", and the new case, where attribute parts are separated by "?", so I tried to keep that logic separated.

[the remark this answers is not in the document]
Comment (fashxp) [reply src/Controller/AdminController.php]: :+1:

**Comment by fashxp** [comment]
thx

**Pull request merged**